content/2_Bronze/Rect_Geo.mdx

```diff
@@ -78,18 +78,47 @@ int main() {
 	int a, b, c, d;
 	cin >> a >> b >> c >> d;
 
-	int ans = 0;
-	ans += b-a; // add length of first interval
-	ans += d-c; // add length of second interval
-	// if intervals intersect, subtract the length of the intersection
-	ans -= max(min(b, d)-max(a, c), 0);
+	int union = (b - a) + (d - c);  // the sum of the two intervals
```

Comment:
The terminology used here is incorrect. This isn't a union; the union is the final answer. I'm not even sure what to call this intermediate value, I think "total" is more suitable, as in "total paint coats."